python/GafferUI/WidgetAlgo.py

```diff
@@ -48,3 +54,14 @@ def joinEdges( listContainer ) :
 	for i in range( 0, l ) :
 		visibleWidgets[i]._qtWidget().setProperty( lowProperty, i > 0 )
 		visibleWidgets[i]._qtWidget().setProperty( highProperty, i < l - 1 )
+
+def grab( widget, imagePath ) :
+
+	GafferUI.EventLoop.waitForIdle()
```

Comment:
I'd argue its pretty frustrating to take a screen grab and have it be utter garbage, so if I remove it, then every single call site for `WidgetAlgo.grab()` is going to create its own pause anyways... Unless someone is taking a grab of the exact same thing over and over again?

Would you be ok with it being an optional arg which defaults true?